code/script.js

```diff
@@ -12,36 +54,308 @@ const showMessage = (message, sender) => {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="./assets/chatbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+   `
   }
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+//input from user
+const handleResponse = (response) => {
+  showMessage(response, 'user');
+}
+
+// Bot chat messages
+// .0
+const nights = (fName) => {
+  questionNumber = 1;
+  showMessage(`Hi ${fName}, how many nights do you want to book?`, 'bot');
+}
+
+// .1
+const persons = (nights) => {
+  if (Number(nights) > 0 && Number(nights) <= 28) {
+    showMessage(`How many are going to travel?`, 'bot');
+    questionNumber++
+    bookingSentence += `We're booking a ${nights} day/s trip`
+  } else {
+    showMessage(`That is not a valid number, try again.`, 'bot');
+  }
+}
+
+// .2 with buttons instead of text input
+const where = (persons) => {
+  if (Number(persons) > 0 && Number(persons) <= 14) {
+    questionNumber++
+    bookingSentence += ` for ${persons} person/s`
+    showMessage(`What settings are you looking for?`, 'bot');
+
+    inputWrapper.innerHTML = `
+      <div class="where-btn-wrapper">
+        <div class="where-btn" id="mountain">
+          <img src="./assets/mountains.png" alt="mountains" />
+        </div>
+        <div class="where-btn" id="beach">
+          <img src="./assets/beach.png" alt="beach" />
+        </div>
+        <div class="where-btn" id="city">
+          <img src="./assets/city.png" alt="city" />
+        </div>      
+      </div>
+    `
+    const mountain = document.getElementById('mountain')
+    const beach = document.getElementById('beach')
+    const city = document.getElementById('city')
+
+    mountain.addEventListener('click', () => {
+      nxtQuestion('mountain')
+    })
+    beach.addEventListener('click', () => {
+      nxtQuestion('beach')
+    })
+    city.addEventListener('click', () => {
+      nxtQuestion('city')
+    })
+  } else {
+    showMessage(`That is not a valid number, try again.`, 'bot');
+  }
+}
+
+// .3 after choosing mountain/beach/city - a list to select 
+const place = (place) => {
+  questionNumber++
+  showMessage(`Where do you want to go?`, 'bot');
+  if (place === 'mountain') {
+    inputWrapper.innerHTML = `
+      <select class="select-mountain" id="select">
+        <option selected disabled>
+          Choose a destination
+        </option>
+        <option value="Schweiz">
+          Schweiz
+        </option>
+        <option value="Chile">
+          Chile
+        </option>
+        <option value="Tibet">
+          Tibet
+        </option>
+      </select>
+    `
```

Comment:
You used this code a few times. Try to create a more generic function, that you will be able to use multiple times wherever it will be needed 😸